frontend/src/concepts/areas/const.ts

```diff
@@ -46,6 +46,9 @@ export const SupportedAreasStateMap: SupportedAreasState = {
     featureFlags: ['disableCustomServingRuntimes'],
     reliantAreas: [SupportedArea.MODEL_SERVING],
   },
+  [SupportedArea.CONNECTION_TYPES]: {
```

Comment:
In a previous PR with the feature flag, there is `[SupportedArea.DATA_CONNECTION_TYPES]` that already exists on line 130. You can remove the reference of that and only leave this one, or vice versa. Same for in `frontend/src/concepts/areas/types.ts`

Reply:
Thanks @emilys314, I've updated the other instances to use `CONNECTION_TYPES` as I feel it is more consistent with the other area naming conventions.